docs/content/guides/dagster/enriching-with-software-defined-assets.mdx

```diff
@@ -0,0 +1,407 @@
+
+Preemptively declaring assets offers distinct advantages, including:
+
+### Lineage
+
+As software-defined assets know what other assets they depend on, an asset's lineage can be [viewed easily in Dagit](/concepts/assets/software-defined-assets#viewing-assets-in-dagit).
+
+Assets help track and define cross-job dependencies. For example, when viewing a job that materializes assets, you can navigate to the jobs that produce the assets that it depends on. Additionally, when an upstream asset has been updated more recently than a downstream asset, Dagster will indicate that the downstream asset might be out of date.
+
+### Direct operation
+
+Using software-defined assets enables you to directly operate them in Dagit. On the [Assets page](/concepts/dagit/dagit#assets), you can:
+
+- View the materialization history of an asset
+- Check when the next materialization will occur
+- Launch a run to materialize the asset
+
+Drilling down to the [Asset Details page](/concepts/dagit/dagit#asset-details) for an individual asset shows the sensors or schedules for any jobs targeting the asset. It also includes a **(Re)Materialize** button, allowing you to launch a run that rematerializes the asset, including its ancestors or descendants.
+
+### Improved code ergonomics
+
+Without software-defined assets, you're often forced to choose between:
+
+- **Easy dependency tracking** by containing everything in a single mega-job, which requires maintaining a huge, unwieldy object.
+
+- **Manageable maintenance** by splitting up your pipeline into smaller jobs, which makes tracking cross-job dependencies difficult.
+
+This isn't the case with software-defined assets:
+
+- **Assets track their dependencies,** so you don't need to define mega-jobs just to keep track. This allows you to avoid interruptions in dependency graphs and eliminate the need for root input managers.
+
+- **Using software-defined assets usually means writing less code**. Specifying the inputs to a software-defined asset defines the assets it depends on. This means you don't need to use `@graph` and `@job` to wire dependencies between ops.
+
+  This approach reduces the number of times an asset's name appears in your codebase by half, making improving your codebase's scalability. Refer to the [IO manager-based example](#materialize-two-interdependent-tables-without-an-io-manager) below to see this in action.
+
+---
+
+## When is it appropriate to use software-defined assets?
+
+You should use software-defined assets when:
+
+- You’re using Dagster to produce or maintain assets, AND
+- You know what those assets will be before you launch any runs. Note that using software-defined assets in one job doesn’t mean they must be used in all your jobs.
```

Comment:
"Note that using software-defined assets in one job doesn’t mean they must be used in all your jobs." is a separate point that I don't think is related to this bullet.